builtin/commit.c

```diff
@@ -184,10 +188,13 @@ static void determine_whence(struct wt_status *s)
 		whence = FROM_MERGE;
 	else if (file_exists(git_path_cherry_pick_head(the_repository))) {
 		whence = FROM_CHERRY_PICK;
-		if (file_exists(git_path_seq_dir()))
-			sequencer_in_use = 1;
 		if (file_exists(git_path_rebase_merge_dir()))
-			rebase_in_progress = 1;
+			pick_state = REBASE;
+		else if (file_exists(git_path_seq_dir()))
+			pick_state = MULTI_PICK;
+		else
+			pick_state = SINGLE_PICK;
+		}
```

Comment:
Isn't this curly bracket an extra one? The build seems to be failing because of it: https://dev.azure.com/gitgitgadget/git/_build/results?buildId=27009&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=ce91d5d6-0c55-50f5-8ab9-6695c03ab102

Reply:
Interesting.. How'd that get in there? And how come the make all tests succeeded. Hrm.. Will investigate. Sorry!

Reply:
Updated. I'm guessing my machine was caching the build in some way, I cleaned and verified this time.

Reply:
Excellent!